ui/app/src/services/configuration.ts

```diff
@@ -95,6 +112,54 @@ export function getPreviewToolsConfig(site: string): Observable<PreviewToolsConf
   );
 }
 
+export function getAudiencesPanelConfig(site: string): Observable<AudiencesPanelConfig> {
```

Comment:
Clean up the parsed object. It needn’t follow the legacy format. The idea is for it to be nice, clean and parsed ready for use.